index.js

```diff
@@ -44,52 +78,119 @@ async function checkUrl(url){
     return true;
 }
 
-
 // Create an event listener for messages
 client.on('message', async (message) => {
-    const prefix = '!';
     if (!message.content.startsWith('!') || message.author.bot) return;
+    if (client.user.id === message.author.id) {return}
+    if (message.author.client == true) return;
 
     var args = message.content.slice(prefix.length).split(' ');
     var command = args.shift().toLowerCase();
     args = args.splice(0,1);
-    if(command == 'ping'){
-        message.channel.send('STOP PINGING ME!!!');
-    }
-
-    if(command == 'user'){
-        var response = await uuid(args[0]);
-        var cape_urls = config.cape_urls;
+    try {
+        if (command == `help`) {
+            var randomColor = Math.floor(Math.random()*16777215).toString(16);
+            var sender = message.member.user;
+
+            var DMreply = await embed(`List of commands`, `\`${prefix}help\` - Helps you\n\`${prefix}ping\` - Pings the bot\n\`${prefix}user (mc-username)\` - Gets info about the user`, randomColor)
+            sender.createDM().then (dmChannel => {
+                dmChannel.send(DMreply)
+            }).catch (err => {
+                message.channel.send(`I was unable to finish this command, make sure your DMs are open to all server members. \`ERR: 0xCR0003\``)
+                return
+            });
+            var reply = await embed(`:mailbox_with_mail:\`You got mail!\``, `${new Date().getTime() - message.createdTimestamp}ms`, randomColor)
+            message.channel.send(reply);
+        }
```

Comment:
This will need to be removed or replaced with a different command as it conflicts with other bots.

Reply:
How so?

Reply:
It conflicts with the MEE6 bot and its !help command.

Reply:
Could just change the prefix to something else.

Reply:
Or I got a better idea.